src/Models/IoTWorkspaceProject.ts

```diff
@@ -185,14 +185,38 @@ export class IoTWorkspaceProject extends IoTWorkbenchProjectBase {
         createTimeScaffoldType, openInNewWindow, OpenScenario.createNewProject);
   }
 
-  async openProject(
-      scaffoldType: ScaffoldType, openInNewWindow: boolean,
-      openScenario: OpenScenario): Promise<void> {
+  /**
+   * Validate workspace config file exists.
+   * Handles situation when workspace config file has a different name.
+   * @param scaffoldType
+   */
+  async validateWorkspaceConfigFilePath(scaffoldType: ScaffoldType):

```

Comment:
It seems this function is try to assign workspace file to variable this.workspaceConfigFilePath? The logic is to find files with specified suffix in folder. It throws error if there is no file found, what if there are multiple file found?
BTW, you can use this method (vscode.workspace.findFiles) to find files.

Reply:
If there is many, we use the first file found.